worker/uniter/uniter_test.go

```diff
@@ -204,6 +206,21 @@ func (s *UniterSuite) TestUniterInstallHook(c *gc.C) {
 	})
 }
 
+func (s *UniterSuite) TestUniterUpdateStatusHook(c *gc.C) {
```

Comment:
Is this sufficient? The status update hook also fires after significant events eg config-changed hook etc. Perhaps there are other tests I've forgotten about.